.github/actions/context/action.yaml

```diff
@@ -0,0 +1,111 @@
+name: get_context
+description: 'Get the current context'
+outputs:
+  cosign_version: 
+    description: "Cosign version used for building Connaisseur image"
+    value: ${{ steps.get_context.outputs.COSIGN_VERSION }}
+  chart_version: 
+    description: "Connaisseur Helm chart version"
+    value: ${{ steps.get_context.outputs.CHART_VERSION }}
+  original_repo:
+    description: "Public Connaisseur repo (w/o registry)"
+    value: ${{ steps.get_context.outputs.ORIGINAL_REPO }}
+  original_tag:
+    description: "Current Connaisseur tag, i.e. version"
+    value: ${{ steps.get_context.outputs.ORIGINAL_TAG }}
+  original_image:
+    description: "Full Connaisseur image reference, i.e. repository + tag"
+    value: ${{ steps.get_context.outputs.ORIGINAL_IMAGE }}
+  build_repo:
+    description: "Workflow build repository used for testing"
+    value: ${{ steps.get_context.outputs.BUILD_REPO }}
+  build_tag:
+    description: "Workflow build tag used for testing (unique for each run)"
+    value: ${{ steps.get_context.outputs.BUILD_TAG }}
+  build_image:
+    description: "Workflow build image used for testing, i.e. repository + tag"
+    value: ${{ steps.get_context.outputs.BUILD_IMAGE }}
+runs:
+  using: "composite"
+  steps:
+    - name: Get chart version
+      id: get_chart_version
+      uses: mikefarah/yq@master
+      with:
+        cmd: yq '.version' helm/Chart.yaml
+    - name: Get original image
+      id: get_original_image
+      uses: mikefarah/yq@master
+      with:
+        cmd: yq '.deployment.image' helm/values.yaml
+    - name: Get context
+      id: get_context
+      run: |
+        CHART_VERSION=${{ steps.get_chart_version.outputs.result }}
+        COSIGN_VERSION=$(grep -Eo '^COSIGN_VERSION = .*' Makefile | grep -Eo '[0-9]+\.[0-9]+\.[0-9]+')
+        ORIGINAL_IMAGE=${{ steps.get_original_image.outputs.result }}
+        ORIGINAL_REPO=$(echo "${ORIGINAL_IMAGE}" | cut -d ":" -f 1)
+        ORIGINAL_TAG=$(echo "${ORIGINAL_IMAGE}" | cut -d ':' -f 2)
+        BUILD_REPO=ghcr.io/${{ github.repository }}
+
+        if [ "${{ github.ref_name }}" = "master" ]; then
+          BUILD_TAG="${ORIGINAL_TAG}"
+          if [ "${{ github.workflow }}" = "nightly-scans" ]; then
+            BUILD_TAG=${BUILD_TAG}-nightly
+          fi
+        elif [ "${{ github.ref_name }}" = "develop" ]; then
+          BUILD_TAG="develop"
+        else
+          #TODO: just for test
```

Comment:
remaining TODO

Reply:
yes, I added it to run all workflows again. Sorry, that was my rebase